engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/spel/SpelExpression.scala

```diff
@@ -120,6 +122,27 @@ class SpelExpression(parsed: ParsedSpelExpression,
   }
 }
 
+object NullExpression {
+  def apply(original: String, expectedReturnType: TypingResult, flavour: Flavour): NullExpression =
+    new NullExpression(original, returnedValue(expectedReturnType), flavour)
+
+  def returnedValue(expectedReturnType: TypingResult): Any =
+    expectedReturnType.asInstanceOf[TypedClass].klass match {
```

Comment:
We can't use this `asInstanceOf[TypedClass]`. If someone declare parameter as `java.lang.Object` u will have `Unknown` class here. Maybe more safe wil be `expectedReturnType match {`. and for all unknown situation we will return null. WDYT?

Reply:
Indeed, it failed. Done